### ISSUE TITLE
Autoincrement is only allowed on a public or package-private field

### ISSUE THREAD

**Issue by surfer190** [Raizlabs/DBFlow#298]
Autoincrement is only allowed on a public or package-private field

**Comment by agrosner** [comment]
@surfer190 can you specify what you'd like to do so I can get an understanding of the use case?

**Comment by surfer190** [comment]
@agrosner 
Can't use :

```
@Column @PrimaryKey private long ID;
```

Must use:

```
@Column @PrimaryKey long ID;
```

**Comment by agrosner** [comment]
Fixed in 2.2.1
